### PR TITLE
Bug/cast ids to string

### LINKED ISSUE #30: [Documentation] Move badges below H1 heading
*Copied from [fivetran/dbt_tiktok_ads_source #24](https://github.com/fivetran/dbt_tiktok_ads_source/issues/24).*

### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

We need to move the badges at the top of the README to be below the H1 heading. This is to be consistent with other popular README formats and also ensure there is no relevant information above the H1 heading. Additionally, we will need to adjust the orientation of the badges to be consistent with the rest of the README which is left-oriented.

### How would you implement this feature?

Move the badge information below the H1 heading and change the `<p align=` to be `"left"`.

### For Example
In the before changes codeblock below, the H1 header (identified by a single `#`) needs to be moved to the very top. Then we must ensure everything within the `<p>` tag is below the H1 header, but before the first H2 heading (identified by a double `##`).
> Note: This example uses TikTok Ads Source. These changes should be applied consistently across all relevant repositories, using the repository-specific badge information. 

#### Before Changes
```
<p align="center">
    <a alt="License"
        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
    <a alt="dbt-core">
        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
    <a alt="Maintained?">
        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
</p>

# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))
## What does this dbt package do?
```

#### After Changes
Notice the `<p align="left">` update as well as the location of the badges between the H1 and H2 header with appropriate spacing.
```
# Tiktok Ads Source dbt Package ([Docs](https://fivetran.github.io/dbt_tiktok_ads_source/))

<p align="left">
    <a alt="License"
        href="https://github.com/fivetran/dbt_tiktok_ads_source/blob/main/LICENSE">
        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
    <a alt="dbt-core">
        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.3.0_<2.0.0-orange.svg" /></a>
    <a alt="Maintained?">
        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
</p>

## What does this dbt package do?
```

### Are you interested in contributing this feature?

- [ ] Yes.
- [ ] Yes, but I will need assistance.
- [ ] No.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
## PR Overview
**This PR will address the following Issue/Feature:**
- #30 
- https://github.com/fivetran/dbt_greenhouse/issues/26

**This PR will result in the following new package version:**

- v0.8.0 because of casting changes.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

>  ## Breaking Changes
> - Casted `*_id` fields as strings to ensure compatibility in joins downstream.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] ~dbt run (if incremental models are present) && dbt test~

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

- see [transformation PR](https://github.com/fivetran/dbt_greenhouse/pull/28)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

**Comment by fivetran-catfritz** [comment]
While only a couple fields generated the issue's error in our seed data, I went ahead and casted all the ids since there were lots of joins involving ids, and I wanted to cover all bases to be safe.

File: models/stg_greenhouse__eeoc.sql
Comment (fivetran-joemarkiewicz) [inline models/stg_greenhouse__eeoc.sql]: Should we cast this `veteran_status_id` field as well?

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md]: When reviewing the `*_id` fields original datatypes it seems the majority are integers ([example](https://github.com/fivetran/dbt_greenhouse_source/blob/ab89d72f57ec4be3f51299fbd3f89bc9e5f01f40/macros/get_scheduled_interviewer_columns.sql#L5-L7)). I'm on board with switching to string for more flexibility. However, I feel we should call this out in the CHANGELOG. Can you add a sub bullet here that says the majority of these fields were integers previously.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
@fivetran-catfritz thanks for working through these changes! Overall this PR is looking great. I just have a few comments and requests before this is ready for approval. Let me know once this is ready for re-review.

File: models/stg_greenhouse__eeoc.sql
Comment (fivetran-catfritz) [reply models/stg_greenhouse__eeoc.sql]: yes, thanks. updated.

File: CHANGELOG.md
Comment (fivetran-catfritz) [reply CHANGELOG.md]: updated!

**Review by fivetran-catfritz — COMMENTED** [review]
@fivetran-joemarkiewicz thanks--I have updated!

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: CHANGELOG.md
Comment (fivetran-avinash) [inline CHANGELOG.md]:
```suggestion
- Updated `*_id` fields in all staging models to be cast as strings to ensure compatibility in downstream joins by avoiding potential type mismatches.
```

File: models/stg_greenhouse__job_opening.sql
Comment (fivetran-avinash) [inline models/stg_greenhouse__job_opening.sql:29]: why was `job_openining_id` originally spelled wrong 😅 

I guess we should leave it to avoid making field changes too.

File: models/stg_greenhouse__scheduled_interview.sql
Comment (fivetran-avinash) [inline models/stg_greenhouse__scheduled_interview.sql]:
```suggestion
        cast(id as {{ dbt.type_string() }}) as scheduled_interview_id,
```

File: CHANGELOG.md
Comment (fivetran-avinash) [inline CHANGELOG.md:12]: Is this just an integration test? It's a run on integration test data so it's also executing the model too. Maybe we could say something like "Added additional CI/CD run and test scripts for when all `greenhouse_using_*` variables are disabled."

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-catfritz A few questions before approval (sorry about the formatting of the review, looks like it separated it out to above and below this comment).

File: CHANGELOG.md
Comment (fivetran-catfritz) [reply CHANGELOG.md:12]: I don't understand this comment. What exactly is wrong with saying I added the integration tests? The only people that will care about this line is us.

Comment (fivetran-avinash) [reply CHANGELOG.md:12]: Fair enough.

**Review by fivetran-avinash — APPROVED** [review]

**Pull request merged**
